languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -46469,6 +46469,39 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Grammatik -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammatik">
+        <rule id="HANDELN_UEBER" name="handeln über (von)" default="temp_off">
+            <antipattern>
+                <token inflected="yes">handeln</token>
+                <token min="0" skip="-1"/>
```

Comment:
siehe Kommentare in anderen Commits. `skip="-1"` kommt in das Token, nach dem beliebig viele weitere Token stehen können.